api.planx.uk/modules/saveAndReturn/service/validateSession.ts

```diff
@@ -135,8 +135,8 @@ async function reconcileSessionData({
     }
   };
 
-  // remove all auto-answered breadcrumbs
-  // (auto-answers are reconstructed in the editor by `upcomingCards`)
+  // remove all auto-answered breadcrumbs because their automation may rely on the same data values as an altered node
+  //   (auto-answers can be reconstructed on forwards navigation if they are still auto-answerable on resume)
```

Comment:
See existing `test('auto-answered breadcrumbs with matching "data.val" values are also removed')` in `validateSession.test.ts` to better understand this. I ultimately believe this is _correct_ behavior